sdk/ai/ai-inference-rest/samples/v1-beta/javascript/audioUrlChatCompletion.js

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * Demonstrates how to get chat completions using an audio URL.
+ * NOTE: Audio URL completions currently work only with Phi multimodal models.
```

Comment:
Is there a document in ms learn for limitations or maybe README? Would be better to refer to document than maintain the information here.

NOTE: Audio URL completions currently work only with Phi multimodal models.

Reply:
[Updated with support matrix](https://github.com/Azure/azure-sdk-for-js/pull/33359/commits/591a4aa3c65b2c0f17690b468523d62aee9b9662)